styles.css

```diff
@@ -408,3 +413,194 @@ footer textarea {
   padding-left: 10px;
   padding-top: 10px;
 }
+
+@media screen and (min-width: 376px) {
```

Comment:
Kindly ensure that the mobile version remains responsive until the desktop screen size (768px). This means that the value for the min-width property should be 768px or above. This screen sizes below are supposed to display as mobile.

Your design below 768px:
![image](https://user-images.githubusercontent.com/20554583/208026765-d12b08a1-0fde-4dc3-a545-e472ac1cf463.png)

Your design at 515px:
![image](https://user-images.githubusercontent.com/20554583/208027030-431b7627-362f-4483-9d8d-2634f5e87f77.png)